etc/p3-config/p3-config.yml

```diff
@@ -451,15 +451,15 @@ p3_images:
   - "{{ p3_image_ubuntu_cuda }}"
   - "{{ p3_image_mlnx_ofed }}"
   - "{{ p3_image_anaconda }}"
-  - "{{ p3_image_fedora_atomic_27 }}"
+  - "{{ p3_image_fedora_atomic }}"
 
 # Fedora Atomic 27 image
-p3_image_fedora_atomic_27:
-  name: "FedoraAtomic27"
-  image_url: "https://download.fedoraproject.org/pub/alt/atomic/stable/Fedora-Atomic-27-20180326.1/CloudImages/x86_64/images/Fedora-Atomic-27-20180326.1.x86_64.qcow2"
+p3_image_fedora_atomic:
+  name: "FedoraAtomic29"
```

Comment:
Thought you decided not to switch to 29?

Reply:
We still use 29 for kubernetes, and swarm will also be able to use 29 with docker-ce installed (probably via ansible rather than manual image configuration as it’s currently done)